### ISSUE TITLE
Macbook Pro 16 2019 starts lagging after 30-40 minutes after app launch

### ISSUE THREAD

**Issue by vbuberen** [crystalidea/macs-fan-control#248]
Last week got a brand new MacBook Pro 16 and installed Macs Fan Control on it right away just like I did for my MacBook Pro 15 (2017) a few years ago.

The strange thing that in about 30 minutes of the app running everything starts to lag and in Activity Monitor I see that Kernel task spikes to enormous CPU usage (see the attached screenshot)
<img width="107" alt="Screenshot 2019-12-09 at 11 28 03" src="https://user-images.githubusercontent.com/13467769/70793277-9e3e9e00-1da3-11ea-89fd-705c3d20c7b6.png">

The only way to resolve this is to reboot the device and not launch Macs Fan Control. Tried to launch nothing except it after reboot, but still running into this issue. Without Macs Fan Control I was able to work full 2 days without any such lags. 

Just tried to reproduce the issue 2 hours ago and was able to see the same behaviour again.

I had access to another MacBook 16 Pro with the same configuration and was able to get the same issue.

It seems that there are some changes in the latest Pros, which require some code changes.

Out of the box the device had Catalina 10.15.1 installed and yesterday I updated to 10.15.2, but the problem is still there.

Both Macs are in default configuration with 2.6 GHz 6 core i7 CPU.

**Comment by kleuter** [comment]
![image](https://user-images.githubusercontent.com/2600624/70795813-f7590280-1da0-11ea-8ed7-ca6331298270.png)
From the [faq](https://www.crystalidea.com/macs-fan-control/faq)

You you'd like to report a bug, please include technical information from the menu "More" -> "Copy technical info". Please do not forget to put technical info inside the details tag:
<details>
YOUR TECH INFO
</details>
Screenshot of the main window of Macs Fan Control is highly appreciated as well.

**Comment by vbuberen** [comment]
Thanks. 

Tried to reset SMC in the very first day after seeing this issue.
Attaching screenshots of the main window as well as of the preset I used for both fans.
Note, the same preset is used on my another Macbook Pro 15, which is 2017 model. It works just fine there.

Switched to `Auto` to see if it fixes the issue.
<img width="782" alt="Screenshot 2019-12-13 at 15 31 45" src="https://user-images.githubusercontent.com/13467769/70803901-c5a26480-1dbd-11ea-8220-7f9ee5f8bba9.png">
<img width="781" alt="Screenshot 2019-12-13 at 15 31 39" src="https://user-images.githubusercontent.com/13467769/70803900-c509ce00-1dbd-11ea-935d-9e296cb1b4a4.png">
